ext/opcache/jit/README.md

```diff
@@ -0,0 +1,28 @@
+Opcache JIT
+===========
+
+This is the implementation of Opcache's JIT (Just-In-Time compiler),
+This converts the PHP Virtual Machine's opcodes into x64/x86 assembly,
+on POSIX platforms and Windows.
+
+It generates native code directly from PHP byte-code and information collected
+by SSA static analysis framework (a part of the opcache optimizer).
```

Comment:
```suggestion
by the SSA static analysis framework (a part of the opcache optimizer).
```

Reply:
Missed that comment - I addressed that in 3e045d5f2b109cb53c01e626f0c0a668cd16a352